DatadogInternal/Sources/Models/SessionReplay/SessionReplayConfiguration.swift

```diff
@@ -17,7 +17,7 @@ public enum SessionReplayPrivacyLevel: String {
     case mask
 
     /// Mask input elements, but record all other content.
-    case maskUserInput = "mask_user_input"
+    case maskUserInput = "mask-user-input"
```

Comment:
It seems that we were passing the wrong privacy level to webviews. See [doc](https://docs.datadoghq.com/real_user_monitoring/session_replay/privacy_options) for ref.

Reply:
Nice catch!!